### PR TITLE
Added Directory.packages.props file for DUI3-DX dir

### PR DESCRIPTION
**Pull request by JR-Morgan**

### CODE REVIEW

File: DUI3-DX/Directory.Packages.props [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageVersion Include="Serilog.Extensions.Logging" Version="7.0.0" />
```
Comment (BovineOx) [inline DUI3-DX/Directory.Packages.props:6]: That feels like the wrong version but I could be wrong?

**Review by BovineOx — CHANGES_REQUESTED** [review]
Just one query but otherwise fine

File: DUI3-DX/Directory.Packages.props (hunk 1 above)
Comment (JR-Morgan) [reply DUI3-DX/Directory.Packages.props:6]: It was the version we were using in the connector projects

**Comment by JR-Morgan** [comment]
I was hoping that we could get away with mixing centrally managed versions, and ones hardcoded in a csproj file
It looks like this isn't the case (see errors in CI, although, locally the build is successful)

Things like RhinoCommon cannot be a centrally managed version, so I'm not sure how we can navigate this.
Perhaps, if we moved some of the shared code into a common connector project, we'll find we avoid referencing the same dependency in multiple places. (And therefore have no need for central package management)

**Review by AlanRynne — COMMENTED** [review]
Centrally managed packages are useful for "widely used packages in our repo", i.e. things that we now 100% need to be the same version in every project and using a different one will result in unexpected behaviour.

I think for most `Microsoft.*` and things like logging it makes sense to centralise and lock it down.

For connectors it's tricky since the nuget's need to be different versions for each host app release, but I do still see this having value.

**Pull request closed without merging**